llvm/test/CodeGen/AArch64/ptrauth-global-no-pic.ll

```diff
@@ -0,0 +1,51 @@
+; RUN: llc -mtriple aarch64-elf --relocation-model=static         -mattr=+pauth -filetype=asm -o - %s | FileCheck %s
+; RUN: llc -mtriple aarch64-elf --relocation-model=dynamic-no-pic -mattr=+pauth -filetype=asm -o - %s | FileCheck %s
```

Comment:
As far as I can see, `-filetype=asm` is the default in `llc`. Furthermore, it is [advised](https://llvm.org/docs/TestingGuide.html#fragile-tests) to pass the input to `opt` via redirection - not sure if it has any relation to `llc` (except for uniformity), but using `< %s` makes `-o -` useless, too.

Reply:
Thanks for suggestion! Fixed in d85f8cfd737cbd5e3b1777c4ae1a6317a1436370